sphinx/util/logging.py

```diff
@@ -65,6 +65,12 @@ def getLogger(name):
     Sphinx logger always uses ``sphinx.*`` namesapce to be independent from
```

Comment:
The first sentence of this docstring should use `:class:` role for cross referencing.

Reply:
Done